webpack.config.js

```diff
@@ -485,13 +485,15 @@ if (DEV) {
     env.PUBLIC_PATH = JSON.stringify(publicPath);
     env.RUDDER_KEY = JSON.stringify(process.env.RUDDER_KEY || '');
     env.RUDDER_DATAPLANE_URL = JSON.stringify(process.env.RUDDER_DATAPLANE_URL || '');
+    env.STRIPE_PUBLIC_KEY = JSON.stringify(process.env.STRIPE_PUBLIC_KEY || '');
```

Comment:
Where are we setting the actual prod values?

Reply:
It gets set in the build environment in the CI repo: https://git.internal.mattermost.com/mattermost/ci/mattermost-webapp/-/merge_requests/22/diffs . I thought it was already pushed, but looks like it was still on my local machine. Pushed now.

Reply:
Now I see it 👍 

We need to make sure to test this change for Cloud too as this has been the same since the launch of cloud and we need to make sure that they are set properly.